src/app/views/public/product/product.less

```diff
@@ -0,0 +1,108 @@
+// product
+.public-dashboard-marketplace-app {
+  padding: 25px 50px 0 50px;
+
+  .public-logo-wrapper {
+    position: relative;
+    display: flex;
+    flex-direction: column;
+
+    .landing-logo {
+      height: 120px;
+      width: 120px;
+    }
+
+    .app-rating {
+      margin-top: 15px;
+    }
+  }
+
+  .public-back-button-container {
+    margin: 30px;
+    margin-top: 0px;
+
+    .public-back-button {
+    border: 1px solid;
+    color: @landing-nav-bg-color;
+    font-size: 16px;
+    font-weight: 200;
+    padding: 15px 40px;
+    vertical-align: middle;
+    text-decoration: none;
+
+      &:hover {
+        background-color: @landing-nav-bg-color;
+        color: @public-main-bg-color;
+        -webkit-transition: all 0.1s ease-in-out;
+        -moz-transition: all 0.1s ease-in-out;
+        -ms-transition: all 0.1s ease-in-out;
+        -o-transition: all 0.1s ease-in-out;
+        transition: all 0.1s ease-in-out;
```

Comment:
leave only the `transition:` attribute, I believe that the others are added when the less file is compiled. @ouranos Do you know if I'm correct on this?